File: fixtures/hub-cobol-clbs-mini/batch/reference_emit_portcomrn.py
"""Reference emit for PORTCOMRN — PORTCOM COPY CRUD EVALUATE → 120."""
# EXPECTED: 120
CMDS = ["CREA", "READ", "UPDT", "DELE"]
CODES = {"CREA": 15, "READ": 25, "UPDT": 35, "DELE": 45}
total = sum(CODES[c] for c in CMDS)
print(f"{total:03d}")

File: fixtures/hub-cobol-clbs-mini/batch/reference_emit_portcomrn.java
// EXPECTED: 120
// Reference emit for PORTCOMRN — PORTCOM COPY CRUD EVALUATE.
import java.util.Map;
public final class PortcomrnRef {
  public static void main(String[] args) {
    String[] cmds = {"CREA", "READ", "UPDT", "DELE"};
    Map<String, Integer> codes = Map.of("CREA", 15, "READ", 25, "UPDT", 35, "DELE", 45);
    int total = 0;
    for (String c : cmds) total += codes.getOrDefault(c, 99);
    System.out.printf("%03d%n", total);
  }
}
